Basim_Muhammad/python_practice/practice.py

```diff
@@ -0,0 +1,128 @@
+# import random
+# for i in range(1,1001):
+# 	if i%2!=0:
+# 		print i 
+
+# i=5
+# while i<1000001:
+# 	print i
+# 	i+=5
+
+
+# a = [1, 2, 5, 10, 255, 3]
+# sum=0
+# for element in a:
+# 	sum+=element
+# print sum 
+
+# a=[1,2,5,10,255,3]
+# sum=0
+# for element in a:
+# 	sum+=element
+# print sum/len(a)
+
+# for num in xrange(1,2001):
+# 	if num%2==0:
+# 		print 'Number is', str(num) + '.This is an even number.'
+# 	else:
+# 		print 'Number is', str(num) + '.This is an odd number.'
+
+# def times_five(arr):
+# 	i=0
+# 	for i in range(0,len(arr)):
+# 		arr[i]=arr[i]*5
+# 	return arr
+
+# result=times_five([2,4,10,16])
+# print result
+
+# def grades():
+# 	i=0
```

Comment:
no need to initialize i at 0.  when you do the for...in range loop it automatically starts i at whatever the 1st argument is